docs/content/en/docs/getting-started/enforcement.md

```diff
@@ -99,57 +125,72 @@ a Sigkill. The successful internal connect is filtered and will not be shown.
 🚀 process default/xwing /usr/bin/curl -s -XPOST deathstar.default.svc.cluster.local/v1/request-landing
 ```
 
-### Enforce File Access Monitoring
+### Enforce file access restrictions
 
 The following extends the example from [File Access Monitoring]({{< ref "docs/getting-started/file-events" >}})
 with enforcement to ensure sensitive files are not read. The policy used is the
-[`file_monitoring_enforce.yaml`](https://github.com/cilium/tetragon/blob/main/examples/quickstart/file_monitoring_enforce.yaml)
-it can be reviewed and extended as needed. The only difference between the
+[`file_monitoring_enforce.yaml`](https://github.com/cilium/tetragon/blob/main/examples/quickstart/file_monitoring_enforce.yaml),
+which you can review and extend as needed. The only difference between the
 observation policy and the enforce policy is the addition of an action block
-to sigkill the application and return an error on the op.
+to `SIGKILL` the application and return an error on the operation.
 
 To apply the policy:
 
 {{< tabpane lang=shell >}}
 
-{{< tab Kubernetes >}}
+{{< tab "Kubernetes (single node)" >}}
+kubectl delete -f https://raw.githubusercontent.com/cilium/tetragon/main/examples/quickstart/file_monitoring.yaml
+kubectl apply -f https://raw.githubusercontent.com/cilium/tetragon/main/examples/quickstart/file_monitoring_enforce.yaml
+{{< /tab >}}
+{{< tab "Kubernetes (multiple nodes)" >}}
 kubectl delete -f https://raw.githubusercontent.com/cilium/tetragon/main/examples/quickstart/file_monitoring.yaml
 kubectl apply -f https://raw.githubusercontent.com/cilium/tetragon/main/examples/quickstart/file_monitoring_enforce.yaml
 {{< /tab >}}
 {{< tab Docker >}}
 wget https://raw.githubusercontent.com/cilium/tetragon/main/examples/quickstart/file_monitoring_enforce.yaml
-docker stop tetragon-container
-docker run --name tetragon-container --rm --pull always \
+docker stop tetragon
+docker run --name tetragon --rm --pull always \
   --pid=host --cgroupns=host --privileged               \
   -v ${PWD}/file_monitoring_enforce.yaml:/etc/tetragon/tetragon.tp.d/file_monitoring_enforce.yaml \
   -v /sys/kernel/btf/vmlinux:/var/lib/tetragon/btf      \
-  quay.io/cilium/tetragon-ci:latest
+  quay.io/cilium/tetragon:latest
```

Comment:
you could make use of `{{< latest-version >}}` if you want here that will inject the latest released version of Tetragon
```suggestion
  quay.io/cilium/tetragon:{{< latest-version >}} 
```